File: function/2.rb
puts("ID를 입력해주세요 : ")
input_id=gets.chomp()

def login(id)
  members=['julie','jsoul','kim']
  for member in members do
    if member==id
      return true
    end
  end
  return false
end

if login(input_id)
  puts('Hello '+input_id)
else
  puts('Who are you?')
end

File: function/2.py
input_id=input("ID를 입력해주세요 : ")

def login(id):
    members=['julie','jsoul','kim']
    for member in members:
        if member == id:
            return True
    return False

if login(input_id):
    print("Hello "+input_id)
else:
    print("Who are you?")
